### DISCUSSION TITLE
intellij idea 中无法拉起登录请求

### DISCUSSION THREAD

**Discussion by maxs-marcel** [community/community#14744] [Copilot Conversations]
您好, 我的idea版本是(Mac M1) 2021.2.2  在login to github时卡住, 无法登录和同意协议. 期待您回复给我解决方案
![image](https://user-images.githubusercontent.com/65930792/162862739-73730533-e27d-40f1-b38b-5229db816057.png)

**Comment by maxs-marcel** [comment]
问题已经得到解决. 我将代理节点切换到美国就可以了.

**Reply by titlog** [reply]
请问你给 idea 配置 sock 代理了吗？我还是不行

**Reply by maxs-marcel** [reply]
我之前的节点是HK, 能正常访问github和google等.  尝试切换到US之后就好了.
没有配置idea代理, 我直接本地代理软件代理的.

**Reply by zArkyd** [reply]
我这也开了科学上网也不行，device code 读条结束后啥也没有弹出，也没任何反应···

**Reply by titlog** [reply]
> 我这也开了科学上网也不行，device code 读条结束后啥也没有弹出，也没任何反应···

开全局代理 idea重启试下

**Comment by imelonkid** [comment]
我的也连不上，全局代理也不行，我家里的电脑是可以的。公司电脑不行
